proto/store/task.proto

```diff
@@ -0,0 +1,69 @@
+syntax = "proto3";
+
+package bytebase.store;
+
+import "store/common.proto";
+import "store/plan_check_run.proto";
+
+option go_package = "generated-go/store";
+
+message TaskPayload {
```

Comment:
Let's see if we can use oneof for TaskDatabaseCreatePayload, etc.

Reply:
I tried oneof, it will generate nested JSON messages after marshal, which requires us DML all task data

Reply:
Thx, that was what I was worried about. Let's do this for now!

Reply:
Alternatively, we can use one single large proto with a union of all fields.

Reply:
I'd like to use `oneof` too, so let's simplify the types of payload messages and maybe migrate to `oneof` later

Reply:
SGTM